### ISSUE TITLE
[Afaceri] Error importing node module

### ISSUE THREAD

**Issue by margelatu** [code4romania/czl-scrape#57]
Ran `npm install`, went smoothly.
Ran `node server/server.js` and got:

```
module.js:457
    throw err;
    ^

Error: Cannot find module 'string'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/imargela/Work/projects/code4ro/czl-scrape/afaceri/server/controllers/contentParser.js:14:14)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/imargela/Work/projects/code4ro/czl-scrape/afaceri/server/server.js:5:21)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
```

npm v3.10.3
node v6.4.0

**Comment by margelatu** [comment]
Upgraded to latest versions of npm (v4.1.2) and node (v7.7.3). Same error, slightly different stacktrace :

```
$ node server/server.js
module.js:472
    throw err;
    ^

Error: Cannot find module 'string'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/imargela/Work/projects/code4ro/czl-scrape/afaceri/server/controllers/contentParser.js:14:14)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/imargela/Work/projects/code4ro/czl-scrape/afaceri/server/server.js:5:21)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

**Comment by abouariu** [comment]
I think I was testing with different string parser libraries and forgot to save the final one to package.json. I'll check and commit changes.

**Comment by margelatu** [comment]
@abouariu : thanks, man

**Comment by abouariu** [comment]
Fixed

**Comment by margelatu** [comment]
Confirmed. Fixed by #61
